src/main/java/com/comphenix/protocol/injector/EntityUtilities.java

```diff
@@ -178,7 +178,9 @@ private Collection<?> getTrackedPlayers(Entity entity) {
 		Validate.notNull(entity, "entity cannot be null");
 
 		Object trackerEntry = this.getEntityTrackerEntry(entity.getWorld(), entity.getEntityId());
-		Validate.notNull(trackerEntry, "Could not find entity trackers for " + entity);
+		if (trackerEntry == null) { //entity.toString() is too expensive to be called every time
+			throw new IllegalArgumentException("Could not find entity trackers for " + entity);
```

Comment:
This is not an illegal argument. I would either use IllegalState or (as it was before) a NullPointerException

Reply:
Emm when I check what is used by `Validate.notNull`, it showed that it was using `IllegalArgumentException`
![image](https://user-images.githubusercontent.com/22975196/230797293-6db523af-2671-4cdb-b8fd-f75885ba8256.png)

Reply:
Ah well then it was wrong before - just looked it up in the commons lang3 code (by accident) - they're using NullPointer there :)
I guess we can keep it like that then

Reply:
i agree it's not strictly an illegal argument, but i'd vote for consistency here in case API users catch this exception